crates/re_renderer/src/wgpu_resources/texture_pool.rs

```diff
@@ -46,6 +46,15 @@ pub struct TextureDesc {
     pub usage: wgpu::TextureUsages,
 }
 
+impl TextureDesc {
+    /// Copies the desc but changes the label.
+    pub fn with_label(&self, label: DebugLabel) -> Self {
```

Comment:
This slipped in here by accident - I need it on my current work-stream but it has nothing to do with the pr description

But now it's here, so I'm keeping it :)